docs/source/users/index.md

```diff
@@ -958,6 +958,19 @@ configuration.
 jupyter lab --AiExtension.allowed_providers=openai --AiExtension.allowed_providers=ai21
 ```
 
+### Chat memory size
+
+This configuration allows for setting the number of chat exchanges the model
+uses as context when generating a response.
+
+One chat exchange corresponds to a user query message and its AI response, which counts as two messages.
+k denotes one chat exchange, i.e., two messages.
+The default value of k is 2, which corresponds to 4 messages.
+
+```
+jupyter lab --AiExtension.default_max_chat_history=4
```

Comment:
Setting the default to `k*2` seems to be unnecessary. `k` should denote one exchange between the human and AI. Also, if we are passing `k*2` it is unclear what happens if the value passed is an odd number, this may cause unexpected issues. 

Need to check if PR #938 which also moves the `/clear` handling to `base.py` has been taken into account in this PR.